content/general/concepts/nft/nft.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'NFT'
+Description: 'Non-fungible token which act like a digital representation to prove a ownership for a digital good backed by blockchain technology.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'NFT'
+  - 'Blockchain'
+CatalogContent:
+  - 'paths/front-end-engineer-career-path'
+  - 'paths/computer-science'
+---
+
+**NFT**, Non-fungible token which act like a digital representation to prove a ownership for a digital good backed by blockchain technology.
+
+The term "token" in the phrase "non-fungible token" refers to this digital representation, and the NFT is just a reference to the original. "Non-fungible" in other word simply means "non-replacable". It can't be replaced, there is only one of them. It is always unique.
```

Comment:
```suggestion
The term "token" in the phrase "non-fungible token" refers to this digital representation. "Non-fungible" simply means "non-replaceable". It can't be replaced, there is only one of them. It is always unique.
```